### Control flow

# load precomputed models (mesh, point cloud, octomap, esdf) or compute them from mesh
load_models: false

# whether the models shall be stored in files in order to load them later
save_models: true

# load a precomputed path (e.g. from the last run) or compute it
load_path: false

# whether the path shall be stored in order to be loaded later
save_path: true

# if path shall be executed or only the precomputations shall be done
execute_path: true


## robot and sensor specifics

# Asterix
#base_frame: "base_link"
#sensor_frame: "spin_lidar_mount_link_fixed" # Note: the sensor frame is the FOV frame of the sensor

#robot_length: 1.0
#robot_width: 0.8


# AICO Spot
base_frame: "base_link"
sensor_frame: "lidar_laser_frame" # Note: the sensor frame is the FOV frame of the sensor

robot_length: 1.1 # (probably need to be increased for execution)
robot_width: 0.5 # (probably need to be increased for execution)
